ci/build.yml

```diff
@@ -59,6 +59,16 @@ jobs:
   - task: PublishTestResults@2
     displayName: Publish test results
     condition: eq(variables['SPEECHSDK_RUN_TESTS'], 'true')
+  - bash: |
+        cd tests/packaging
+        npm install
+        npm install ../../microsoft-cognitiveservices-speech-sdk-$SPEECHSDK_SEMVER2NOMETA.tgz
+        npm run bundle
+        E=$?
```

Comment:
The cd at the end isn't necessary; once the step (process) exits, current directory is gone.
Hence, no need to store the exit code and return it either.
-> drop the last three lines